### ISSUE TITLE
Capability provider logs are not captured by Tui_Logger

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wash-archive#85]
[tui-logger](https://github.com/gin66/tui-logger) has been fantastic for capturing wasmCloud logs and displaying them in an easy to understand and filter format. However, log messages that are logged from inside capability providers are not captured by tui. This also notably includes when actors utilize the `wasmcloud:logging` capability to create log messages.

This is likely being caused by multiple interactions with the `env_logger` crate. [Here](https://github.com/wasmCloud/wash/blob/main/src/up.rs#L426-L247) is where the `tui-logger` is being initialized, which then captures logs that emit from `wash`. This does include `wasmCloud` logs that are emitted from the embedded host.

When running the wasmCloud binary manually, e.g. in the wasmCloud repo:
```
RUST_LOG=info cargo run --release
```
Logs are automatically set to capture at an `info` level, and setting the environment variable `RUST_LOG` **does** capture capability provider logs. You can verify this by running
```shell
RUST_LOG=info wash up
wash ctl start provider wasmcloud.azurecr.io/httpserver:0.10.0 # remove "wash" if in the REPL
```
With the environment variable set, you will see a log printed to the screen inline with the REPL input stating that the HTTPServer capability provider's dispatch has been configured. If you run the command without setting `RUST_LOG`, the message will not be captured at all.

In short, my hypothesis is that capability provider logs are considered somewhat separate modules in terms of a Rust program, so the environment variable `RUST_LOG` enables capturing those modules. Initializing an `env_logger` will only capture logs that directly result from a module running in the current program. The ideal solution to this problem is to find a way to capture **all** logs, especially capability provider logs, and have them displayed in the tui log output.

**Comment by autodidaddict** [comment]
Closing this issue as wash REPL is being deprecated
